doc/architecture-design-cn.md

```diff
@@ -64,6 +65,20 @@ plugins:                        # plugin name list
 
 [返回目录](#目录)
 
+## APISIX对象纵览
+
+![](./images/object-relation.png)
+
+1. Route 是路由，通过定义一些规则来匹配客户端的请求，并加载相应插件以及转发到指定后端 Upstream 。Route 可以等价认为是一个 API 或者匹配规则。
+2. Service 是 API 的组，多个 Route 对应一个 Service 。与 Route 对应关系：n 对 1
```

Comment:
I think `route`,`uptream`, `plugin` have been explained clearly in their own section.